src/test/ui/single-use-lifetime/one-use-in-fn-argument.stderr

```diff
@@ -16,5 +16,37 @@ help: elide the single-use lifetime
 LL | fn a(x: &u32) {
    |    --   --
 
-error: aborting due to previous error
+error: lifetime parameter `'m` only used once
+  --> $DIR/one-use-in-fn-argument.rs:15:11
+   |
+LL | fn center<'m>(_: Single<'m>) {}
+   |           ^^            -- ...is used only here
+   |           |
+   |           this lifetime...
+help: elide the single-use lifetime
+   |
+LL | fn center(_: Single<'_>) {}
+   |         --          ^^
+
+error: lifetime parameter `'y` only used once
+  --> $DIR/one-use-in-fn-argument.rs:17:13
+   |
+LL | fn left<'x, 'y>(foo: Double<'x, 'y>) -> &'x u32 { foo.f }
+   |             ^^ this lifetime... -- ...is used only here
+help: elide the single-use lifetime
```

Comment:
Can you perhaps add a `rustfix` test? I assume this suggestion can be automatically applied, yes?

Reply:
I think it can't due to the multiple ambiguity span bug, unless that has been fixed since Berlin.